feat: Add logs command for journalctl (#186)
samply/bridgehead: README.md

curl -k https://localhost
```

> If you get errors when you do this, you can inspect the logs of your landing page container in order to determine what is going wrong. To do this you can use `./bridgehead docker-logs <Project> landing -f` to follow the logs of the container. This transaltes to a docker compose logs command meaning all the ususal docker logs flags work.

Reviewer: Why landingpage? It should be 
```suggestion
Should the landing page not show anything, you can inspect the logs of the containers to determine what is going wrong. To do this you can use `./bridgehead docker-logs <Project> -f` to follow the logs of the container. This transaltes to a docker compose logs command meaning all the ususal docker logs flags work.
```
